corehq/apps/domain/decorators.py

```diff
@@ -671,13 +670,6 @@ def _inner(request, *args, **kwargs):
 
 def check_domain_migration(view_func):
     def wrapped_view(request, domain, *args, **kwargs):
-        domain_obj = Domain.get_by_name(domain)
-        if domain_obj.redirect_url:
-            # IMPORTANT!
-            #     We assume that the domain name is the same on both
-            #     environments.
```

Comment:
Should this comment be preserved in the new decorator?

Reply:
We are making more assumptions this time:
* The domain name is the same on both
* The app exists on both

Both of these will be true for a migrated environment. So instead of making the comment longer, I figured I'd take it out, and cover the purpose of this feature in the documentation.

Reply:
[Documentation](https://docs.google.com/document/d/1mQxRmboW3CuOkmklUjf5jypgJcv9VX_e26TEiFh1CyI/edit) updated.